src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,21 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, onTabClick, someID, someContent }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li key={tab.id} className={tab.id === someID ? 'is-active' : ''} data-cy="Tab">
+              <a
+                href={`#${tab.id}`}
+                onClick={() => onTabClick(tab.title, tab.id, tab.content)}
```

Comment:
<img width="785" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/23fd815a-38b7-41fa-809f-49bb09b52439">
not implemented